spec/requests/api/v1/conversations/messages/index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'GET api/v1/conversations/{id}/messages', type: :request do
+  let(:user)            { create(:user) }
+  let!(:conversation)   { create(:conversation_with_messages, users: [user]) }
+
+  before do
+    get api_v1_conversation_messages_path(conversation_id: conversation.id),
```

Comment:
you can pass the conversation object directly to `api_v1_conversation_messages_path`

Reply:
That is exactly what I meant. :)